config/jobs/kubernetes-sigs/sig-windows/sig-windows-sac.yaml

```diff
@@ -101,7 +101,7 @@ periodics:
       - --aksengine-win-binaries
       - --aksengine-deploy-custom-k8s
       # Specific test args
-      - --test_args=--node-os-distro=windows --docker-config-file=$(DOCKER_CONFIG_FILE) --ginkgo.focus=(\[sig-windows\]|\[sig-scheduling\].SchedulerPreemption|\[sig-autoscaling\].\[Feature:HPA\]|\[sig-apps\].CronJob).*(\[Serial\]|\[Slow\])|(\[Serial\]|\[Slow\]).*(\[Conformance\]|\[NodeConformance\]) --ginkgo.skip=\[LinuxOnly\]|device.plugin.for.Windows
+      - --test_args=--node-os-distro=windows --docker-config-file=$(DOCKER_CONFIG_FILE) -prepull-images=true --ginkgo.focus=(\[sig-windows\]|\[sig-scheduling\].SchedulerPreemption|\[sig-autoscaling\].\[Feature:HPA\]|\[sig-apps\].CronJob).*(\[Serial\]|\[Slow\])|(\[Serial\]|\[Slow\]).*(\[Conformance\]|\[NodeConformance\]) --ginkgo.skip=\[LinuxOnly\]|device.plugin.for.Windows
```

Comment:
the rest of flags has double `--` , does it matter?

Reply:
According to the golang flags documentation [1], "One or two minus signs may be used; they are equivalent.". But indeed, how many are used is a bit inconsistent. I've put only one, since that's how many I've put in the previous commit [2], and it worked fine [3]. It was consistent with how many minuses the docker-config-file config option had.

[1] https://pkg.go.dev/flag#hdr-Command_line_flag_syntax
[2] https://github.com/kubernetes/test-infra/pull/24500
[3] https://github.com/kubernetes/kubernetes/issues/102100#issuecomment-982659136

Reply:
no idea honestly, just was doing the diff , these things always surprise me and doesn't seem to be consistency 
if you tested it then it has to be ok